spec/services/section/new_service_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "spec_helper"
+
+RSpec.describe Section::NewService do
+  it 'returns the manual and a new section' do
+    user = double(:user)
+    section = double(:section)
+    manual = double(:manual, build_section: section)
```

Comment:
Minor: Can we not use instances of real classes for `user`, `section` & `manual` here? Also how hard would it be to rely on the real implementation of `Manual#build_section`...?